### DISCUSSION TITLE
无法在Linux arm64下启动

### DISCUSSION THREAD

**Discussion by yuiflh** [lyswhut/lx-music-desktop#1317] [Q&A]
系统版本：麒麟v10
CPU：Phytium FT2000/4（64位）
arm64版安装上启动无反应，从2.0.0开始就无法使用了，只能退回到1.x版本才可以正常启动
在2.0.3更新日志写到修复了Linux arm64启动无反应问题，可是依然存在。

求修复！

**Comment by lyswhut** [comment]
使用终端启动程序，附上终端输出日志
